index.html

```diff
@@ -232,6 +232,27 @@ <h4 class="connect-link padding">LET'S CONNECT</h4>
 
       </section>
     </section>
+    <section class="contact-form-section">
+      <h2 class="contact-me-title">Contact me</h2>
+      <p class="contact-description">If you have an application you are interested in developing,
+        a feature that you need built or a project that needs
+        coding. I’d love to help with it</p>
+      <form action="https://formspree.io/f/xbjeokzq" method="post" class="contact-form">
+        <ul>
+          <li>
+            <input type="text" name="name" id="yourname" placeholder="Yeremias" required maxlength="30">
+          </li>
+          <li>
+            <input type="email" name="email" id="youremail" placeholder="natajayanj14@gmail.com" required>
+          </li>
+          <li>
+            <textarea name="message" id="yourmessage" cols="" rows="5" placeholder="Write your message here" required maxlength="500"></textarea>
+          </li>
+        </ul>
+    
+        <button type="submit">Get in touch</button>
```

Comment:
- [ ] Please implement the button interaction states as shown in the mage below.
![btns](https://user-images.githubusercontent.com/99656687/220342487-d5b0c560-12b8-4b0c-8838-069dff9475a6.png)